### DISCUSSION TITLE
카카오맵에서 잔여백신 예약으로 넘어가질 않습니다

### DISCUSSION THREAD

**Discussion by tony9959** [changdoc/naver-vaccine-macro#225] [Q&A]
![캡처](https://user-images.githubusercontent.com/5236487/128140438-b8ee27f3-cf11-405c-8510-9b1d5e449cd2.PNG)
네이버맵과 카카오맵 병행, 카카오맵만 가동 이런식으로 두개 테스트 해 봤었는데요
네이버 백신은 늦어서 실패하더라도 예약창으로 넘어가는 반면 카카오맵은 아무런 소식이 없었습니다
카카오 지갑으로 알림 온거 확인해도 아무런 반응이 없었고요.. 위치도 병원목록 불러온 위치와 똑같은 위치상에서 설정했습니다.
F12 눌러봐도 매크로는 제대로 작동중인거 같은데 뭐가 문제일까요? 일단 최신버전으로 설치 했었습니다.

**Comment by changdoc** [comment]
개발자 도구에서 Console 창에는 별다른 메시지가 없을까요?

생성한 병원 목록에 카카오맵에서 발견된 병원이 검색이 안되면 로그가 노출되게 되어 있습니다.

**Reply by iwantvaccine** [reply]
저도 네이버에서 예약창으로 넘어갔는데 카카오로는 안넘어간거같아요
Console 들어가면
[TIARA] page {page: "map", from: undefined, section: "잔여백신탭", name: "잔여백신지도"}
map_kakao.js:194 total: 36
manifest.json:1 Manifest: property 'start_url' ignored, should be same origin as document.
3325map_kakao.js:194 total: 36

이렇게 나오는데 문제가 있는걸까요?

**Reply by rladbssam317** [reply]
저도 똑같은 현상입니드
